src/capture/image/image_capturer.cpp

```diff
@@ -76,10 +76,9 @@ void ImageCapturer::ConfigureGroupedMode(const std::string& prefs)
 		} else if (format == "rendered") {
 			grouped_mode.wants_rendered = true;
 		} else {
-			LOG_WARNING("CAPTURE: Invalid image capture format specified for "
-			            "'default_image_capture_formats': '%s'. "
+			LOG_WARNING("CAPTURE: Invalid 'default_image_capture_formats' setting: '%s'. "
 			            "Valid formats are 'raw', 'upscaled', and 'rendered'; "
-			            "defaulting to 'upscaled'.",
+			            "using to 'upscaled'.",
```

Comment:
using to 'upscaled' -> using 'upscaled'?

Reply:
fixed